network/message/authorization.go

```diff
@@ -15,6 +15,9 @@ type ChannelAuthConfig struct {
 
 	// AllowedUnicast indicates whether this message is allowed to be sent on the channel via unicast
 	AllowedUnicast bool
+
+	// AllowedPublish indicates whether this message is allowed to be sent on the channel via publish/multicast
+	AllowedPublish bool
```

Comment:
Looking over the message configs, only the test messages support both unicast and publish. Instead of 2 separate configs, how about we use a single one?

Reply:
Yes I noticed that as well, I intentionally added AllowedPublish so that it explicitly lets the developer know that some messages are allowed unicast/publish. If you think it's too verbose we can remove it. Without it, you cannot infer that a unicast message is not allowed to be published without going and looking at the validator code.

Reply:
https://github.com/onflow/flow-go/pull/3525/commits/116aaf544ad672ff1c22c1bea32248e47a8c6c61